### PR TITLE
Allow adding old inputs values

### PR DESCRIPTION
**Pull request by emmanuelgautier**

### CODE REVIEW

File: src/Illuminate/Session/Store.php [hunk 1]
```diff
@@ -372,6 +372,22 @@ public function set($name, $value)
     }
 
     /**
+     * Set the specified item to the old input array.
+     *
+     * @param  string  $key
+     * @param  mixed   $value
+     * @return mixed
```
Comment (lucasmichot) [inline src/Illuminate/Session/Store.php]: This does not return anything, so `@return void` would be better

**Comment by lucasmichot** [comment]
Why not `5.2` ?

File: src/Illuminate/Session/Store.php (hunk 1 above)
Comment (emmanuelgautier) [inline src/Illuminate/Session/Store.php]: It is fixed. Thanks

**Comment by emmanuelgautier** [comment]
Why not in master ? :) . It should be better to commit in 5.2 effectively.

**Comment by lucasmichot** [comment]
Because this feature is backward compatible: https://laravel.com/docs/5.2/contributions#which-branch

**Pull request closed without merging**